src/_helpers/fonts.js

```diff
@@ -1,5 +1,14 @@
-const getLineHeight = (theme, size, lineHeight) => {
-  return parseFloat(theme.fonts.lineHeight[lineHeight]) * parseFloat(theme.fonts.size[size]);
+const getDownwardEvenNumber = (val) => {
+  if (val % 2 === 0) {
+    return `${val}px`;
+  } else {
+    return getDownwardEvenNumber(val - 1);
+  }
+};
+
+const getLineHeight = (fontSize, lineHeightMultiplier) => {
```

Comment:
So we have a 1.4 multiplier. we can get rid of all this 😆

Reply:
I can't see those changes on figma. 1.4 multiplier for everything?